src/sentry/static/sentry/app/views/dashboardsV2/widgetCard.tsx

```diff
@@ -93,6 +100,51 @@ class WidgetCard extends React.Component<Props> {
     );
   }
 
+  renderContextMenu() {
+    const {widget, selection, organization, showContextMenu} = this.props;
+
+    if (!showContextMenu) {
+      return null;
+    }
+
+    const menuOptions: React.ReactNode[] = [];
+
+    if (
+      widget.displayType === 'table' &&
+      organization.features.includes('discover-basic')
+    ) {
+      // Open table widget in Discover
+
+      if (widget.queries.length) {
+        const query = widget.queries[0];
+
+        const eventView = eventViewFromWidget(widget.title, query, selection);
+
+        menuOptions.push(
+          <MenuItem
+            key="open-discover"
+            onClick={event => {
+              event.preventDefault();
+              browserHistory.push(eventView.getResultsViewUrlTarget(organization.slug));
+            }}
+          >
+            {t('Open in Discover')}
+          </MenuItem>
+        );
+      }
+    }
+
+    if (!menuOptions.length) {
+      return null;
+    }
+
+    return (
+      <div style={{marginLeft: space(1)}}>
```

Comment:
Should this be a styled component?

Reply:
@markstory I converted this into a styled component.